src/taipy/_cli/_scaffold_cli.py

```diff
@@ -15,23 +15,22 @@
 from cookiecutter.main import cookiecutter
 
 from taipy._cli._base_cli import _CLI
+from taipy.logger._taipy_logger import _TaipyLogger
 
 
 class _ScaffoldCLI:
+    __logger = _TaipyLogger._get_logger()
     __TAIPY_PATH = pathlib.Path(__file__).parent.parent.resolve() / "templates"
 
-    _TEMPLATE_MAP = {
-        "default": str(__TAIPY_PATH / "taipy-default-template"),
-        "multi-page-gui": str(__TAIPY_PATH / "multi-page-gui"),
-    }
+    _TEMPLATE_MAP = {str(x.name): str(x) for x in __TAIPY_PATH.iterdir() if x.is_dir()}
 
     @classmethod
     def create_parser(cls):
         create_parser = _CLI._add_subparser("create", help="Create a new Taipy application.")
         create_parser.add_argument(
             "--template",
-            choices=["default", "multi-page-gui"],
-            default="default",
+            choices=list(cls._TEMPLATE_MAP.keys()),
+            default="taipy-default-template",
```

Comment:
The default choice is "taipy-default-template" for now.

We can rename the template in taipy-templates latter.

Reply:
I guess you meant to rename it to 'default'. Yes. In any case the default template will probably be changed in the next weeks to generate an empty application asking the user more questions like:
- How many pages and what are the page names?
- Do you want to run a Core service?
- a Rest service?